docs/sql-server/sql-server-2012-release-notes.md

```diff
@@ -56,13 +56,15 @@ Before installing [!INCLUDE[ssCurrent](../includes/sscurrent-md.md)], consider t
 ### 1.3 SQL Server Setup might fail while trying to start the SQL Server Browser service  
 **Issue:** SQL Server Setup might fail while trying to start the SQL Server Browser service, with errors similar to the following:  
   
-<pre>The following error has occurred:  
-Service 'SQLBrowser' start request failed. Click 'Retry' to retry the failed action, or click 'Cancel' to cancel this action and continue setup.</pre>  
+```The following error has occurred:  
+Service 'SQLBrowser' start request failed. Click 'Retry' to retry the failed action, or click 'Cancel' to cancel this action and continue setup.
+```
   
 or  
   
-<pre>The following error has occurred:  
-SQL Server Browser configuration for feature 'SQL_Browser_Redist_SqlBrowser_Cpu32' was cancelled by user after a previous installation failure. The last attempted step: Starting the SQL Server Browser service 'SQLBrowser', and waiting for up to '900' seconds for the process to complete.</pre>  
+```The following error has occurred:  
```

Comment:
You have to move the "The following error has occurred:" into the code block. Thanks!

Reply:
Done.